src/Grid/utils.jl

```diff
@@ -340,3 +340,25 @@ for (func,                   entity_f,            entity_t,     filter_f,
     end
 end
 end
+
+
+
+"""
+    bounding_box(grid::AbstractGrid)
+
+Computes the bounding box for a given grid, based on its node coordinates. 
+Returns the minimum and maximum vertices of the bounding box.
```

Comment:
```suggestion
Returns the minimum and maximum vertex coordinates of the bounding box.
```
I think this is a bit more clear.